bin/cron/summer_workshops_to_gdrive

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env ruby
+require_relative '../../lib/cdo/only_one'
+exit unless only_one_running?(__FILE__)
+
+require_relative '../../dashboard/config/environment'
+require 'cdo/google_drive'
+
+# Squash an unhelpful warning that's causing a Honeybadger error
+# see https://github.com/nahi/httpclient/issues/252#issuecomment-302427338
+class WebAgent
+  class Cookie < HTTP::Cookie
+    def domain
+      original_domain
+    end
+  end
+end
+
+#
+# Uses a Google Cloud service account to write summer workshop data for this year into
+# a spreadsheet in Google Drive (with permissions locked down to our organization) for exploration
+# by our programs team.
+#
+
+subjects = [
+  Pd::SharedWorkshopConstants::SUBJECT_SUMMER_WORKSHOP,
+  Pd::SharedWorkshopConstants::SUBJECT_CSF_101,
+  Pd::SharedWorkshopConstants::SUBJECT_CSF_201
+]
+
+workshops = []
+
+# The first row is an array of strings, one for each attribute name.
+workshops << Api::V1::Pd::WorkshopDownloadSerializer.new(Pd::Workshop.first).attributes.keys.map(&:to_s)
+
+# Subsequent rows are for each workshop.
+Pd::Workshop.where(subject: subjects).find_each.each do |w|
```

Comment:
From the example here https://api.rubyonrails.org/classes/ActiveRecord/Batches.html, seems like it is not necessary to do both `find_each` and `each`?

Reply:
Oops, good catch.  That was left over from temporary testing.